Applications/Games/SMITE/Online/script.js

```diff
@@ -0,0 +1,40 @@
+include(["engines", "wine", "quick_script", "online_installer_script"]);
+include(["engines", "wine", "plugins", "windows_version"]);
+include(["engines", "wine", "verbs", "dxvk"]);
+
+// RESEARCH
+/// .NET is mandatory -> mono is mandatory
+/// DXVK?
+
+// TODO : Add steamless option
+// PROTON: https://github.com/ValveSoftware/Proton/issues/2208
+/// TODO: Needs MONO instead of .NET -> Results in fatal on steam using proton in 2019-01-13
+// WINEHQ : https://appdb.winehq.org/objectManager.php?sClass=version&iId=34969&iTestingId=98867
+/// ""I used winetricks to install d3dx10 d3dx11_43 d3dx9_43 directx9 dotnet40 gdiplus vcrun2008 vcrun2010 xact wininet""
+//// gdiplus? xact? dotnet? -> Rest probably irelevant
+
+// Installs SMITE
+
+var installerImplementation = {
+    run: function () {
+        new LocalInstallerScript()
+            .name("SMITE")
+            .editor("Titan Forge Games")
+            .publisher("Hi-Rez Studios")
+            .applicationHomepage("https://www.smitegame.com/")
+            .author("Kreyren")
+            .url("http://cds.q6u4m8x5.hwcdn.net/Installs/InstallSmite.exe") // SOURCE : https://www.smitegame.com/download
+            .checksum("10c12741d19975d3e9094b5ebeaf52cf1096ae45") // Is probably changed often, expected chasum missmatch
```

Comment:
@ImperatorS79 Not tested, rather wait for feedback..

Reply:
Don't put comments containing "waiting for feedback" or "needs to be fixed". As long as these comment exist, the merge request is not mergeable. 
Better use github discussion

Reply:
@qparis Script is in testing, comments will be removed once it's able to run the game on platinum.

Reply:
Please don’t. Script is in testing does not mean that coding convention should be different

Reply:
@qparis The comments are mandatory for me to finish the script i would have to develop the script outside of this merge request -> Limit my workload even more

You sure you want me to remove them?

Reply:
@plata @madoar What do you think? 
In my opinion, merge requests are supposed to be mature enough so that they are ready to be merged. The code review process does not seems suitable to provide support for writing script.  We need to find another channel for this. 
What is your opinion?

Reply:
@qparis My reasoning to initial commits is to make development easier for me so that members can share their tips/commit -> making the script on platinum easier.

Reply:
I'm not entirely sure what we're discussing^^
Here my thoughts about merging script PRs:

In my opinion script PRs should only be merged when they are ready, i.e. working. Otherwise we would have a repository full of defective scripts. A script should only be marked as `testing` if:
- you're not entirely sure it works or 
- it worked previously but stopped working after an update
- it is working on your platform but you can't test it on the other one, but it should also work there

Otherwise, I think we should not merge the PR and wait until it is working. This also gives other people time to take a look at the open PRs and add their suggestions. After a script has been merged it is only one among over a hundred scripts, of which nobody really knows which one currently works, which makes it much harder for other people to add their ideas.

Reply:
Fair enough

Reply:
we should probably open a PR for all scripts that are in testing come to think of it..

Reply:
No, too many PR make discussion impossible. PR is made for code review